File: Back/src/main/java/com/example/buensaborback/domain/entities/ArticuloManufacturado.java
package com.example.buensaborback.domain.entities;

import com.fasterxml.jackson.annotation.JsonManagedReference;
import jakarta.persistence.*;
import lombok.*;
import lombok.experimental.SuperBuilder;

import java.util.ArrayList;
import java.util.HashSet;
import java.util.List;
import java.util.Set;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@ToString
@Entity
@SuperBuilder
public class ArticuloManufacturado extends Articulo{

    private String descripcion;
    private Integer tiempoEstimadoMinutos;
    private String preparacion;
    private boolean eliminado;

    @OneToMany(mappedBy = "articuloManufacturado", fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true)
//    @JoinColumn(name= "articulo_manufacturado_id")
    @Builder.Default
    @JsonManagedReference
    private List<ArticuloManufacturadoDetalle> articuloManufacturadoDetalles = new ArrayList<>();
}

File: Front/buenSaborFront/src/models/ArticuloManufacturado.ts
import ArticuloManufacturadoDetalle from "./ArticuloManufacturadoDetalle";
import Categoria from "./Categoria";
import Imagen from "./Imagen";
import UnidadMedida from "./UnidadMedida";

export default class ArticuloManufacturado{
    id:number = 0;
    denominacion:string = "";
    precioVenta:number = 0;
    descripcion:string = "";
    tiempoEstimadoMinutos:number = 0;
    preparacion:string = "";    
    imagenes:Imagen[] = [];
    unidadMedida:UnidadMedida = new UnidadMedida();
    categoria:Categoria = new Categoria();
    articuloManufacturadoDetalles:ArticuloManufacturadoDetalle[] = [];
    type: string = "articuloManufacturado";
}